src/activity_actor.h

```diff
@@ -79,6 +183,9 @@ class hacking_activity_actor : public activity_actor
         void start( player_activity &act, Character &who ) override;
         void do_turn( player_activity &, Character & ) override {};
         void finish( player_activity &act, Character &who ) override;
+        bool can_resume_with( const activity_actor &, const Character & ) const override {
```

Comment:
Is there a reason this is overriden in each derived class instead of returning false as default in the base class?

Reply:
Not a strong reason. I did this because a  TODO from `player_activity::can_resume_with` said a "pure virtual" `can_resume_with` should be used once `activity_actors` exist.

Reply:
I have changed to default to returning false, thanks again for the feedback!